keras/backend/tensorflow_backend.py

```diff
@@ -2312,22 +2322,28 @@ def __init__(self, inputs, outputs, updates=None, name=None, **session_kwargs):
                     updates_ops.append(update)
             self.updates_op = tf.group(*updates_ops)
         self.name = name
+        # additional tensor substitutions
+        self.feed_dict = session_kwargs.pop('feed_dict', {})
+        # additional operations
+        self.fetches = session_kwargs.pop('fetches', [])
+        if not isinstance(self.fetches, list):
```

Comment:
Same comment as in the other PR: this could be any iterable, and we should normalize it to a list.

Reply:
In the `__call__()` method we construct a list of operations for `fetches` argument. `Session.run()` docs say it can contain an operation, list, tuple, dict or other structure of operations. Via `**session_kwargs` we should be able to pass anything that `run()` accepts. Also we ignore the results of these extra ops, since `__call__()` should return only results of `outputs` ops. Thus it seems to me that a correct behavior might be just to append `fetches` from `**session_kwargs` as an element of our constructed list and not try to convert it to list (ie. unwrap it). Trying to convert any iterable (including dict) to list would be wrong.